app/kumactl/cmd/install/install_transparent_proxy_test.go

```diff
@@ -66,15 +60,15 @@ var _ = Describe("kumactl install transparent proxy", func() {
 			},
 			goldenFile: "install-transparent-proxy.defaults.golden.txt",
 		}),
-		Entry("should generate defaults with user id and DNS redirected and log deprecate", testCase{
+		Entry("should generate defaults with user id and DNS redirected", testCase{
 			extraArgs: []string{
 				"--kuma-dp-uid", "0",
 				"--redirect-all-dns-traffic",
 				"--redirect-dns-port", "12345",
 				"--redirect-dns-upstream-target-chain", "DOCKER_OUTPUT",
 			},
+			errorMessage: "# [WARNING] error occurred when validating if 'conntrack' iptables module is present. Rules for DNS conntrack zone splitting won't be applied: exec: \"iptables\": executable file not found in $PATH\n",
```

Comment:
@lahabana seems still to be failing at this - https://app.circleci.com/pipelines/github/kumahq/kuma/24462/workflows/a8a0de65-c3ae-4de4-8bef-f6403905dad8/jobs/483720/tests#failed-test-0

Reply:
Yeah the error message is different between macOS and Linux. Need to find time to do rework the test

Reply:
I've switched to a `match`